src/chapters/setup-intro-data-types/more-data-types.rst

```diff
@@ -66,72 +115,22 @@ address* of the object, which we visualize as an arrow pointing from the
 variable box to where the data is stored. Instead of holding the actual ``Cat``
 data, ``myCat`` stores *directions* for finding the data in memory.
 
-When we assign a value to a reference type, as in ``Cat sameCat = myCat``, we
-do NOT create a second copy of the object or its data. Instead, we make a
-second arrow pointing to the same memory location.
+When we try to assign ``myCat`` to another variable, as in
+``Cat sameCat = myCat``, we do NOT create a second copy of the object or its
+data. Instead, we make a second arrow pointing to the same memory location.
 
-The distinction between reference types and primitives is important, if
-subtle. As you continue learning Java, you will see that reference types are
+The distinction between object types and primitives is important, if
```

Comment:
to avoid positioning the notion of reference types or reference variables in comparison to the primitive vs object types discussion from earlier, I suggest being more specific with our previous mention of object types and how those instances refer, or point, to the same 'memory instance'. Here's some written down version of my thoughts - 

Strings, for example, have been introduced as an object type. This kind of data is also a Class. When a new String is created, the variable created points to a String object. The fundamental String object is defined by the language. However, other non-primitive, object types that are defined by the programmer, have this same reference variable quality. Eg, Cat Class mentioned on this page.

Reply:
I added a "Java objects" section to lead into this idea, removed "reference type" completely from the page, and moved some of your wording into an admonition box.